frontend/src/components/Lobby/LandingPage.css

```
@@ -0,0 +1,169 @@
+/* src/components/Lobby/LandingPage.css */
+
+.landing-page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  background: linear-gradient(135deg, #1a1a2e 0%, #16213e 50%, #0f3460 100%);
+  color: #fff;
+  font-family:
+    'Inter',
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)